examples/langchain_multimodal_rag.ipynb

```diff
@@ -0,0 +1,366 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7efe0f92-fdbb-4471-b74c-5bdaafed8102",
+   "metadata": {},
+   "source": [
+    "# Multimodal RAG with LangChain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "12e78be0-9abe-4d1a-8aaa-61230b059792",
+   "metadata": {},
+   "source": [
+    "This cookbook shows how to use LangChain to query the table and text extraction results of nv-ingest's pdf extraction tools"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdfad056-baef-448e-b5b3-4544e6b06472",
+   "metadata": {},
+   "source": [
+    "To start we'll need to make sure we have some dependencies installed"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "2dcc96cf-f8e1-4baa-b1ca-b6521d23dec8",
+   "metadata": {},
+   "source": [
+    "pip install langchain langchain_community langchain_chroma langchain-openai"
+   ]
+  },
+  {
```

Comment:
Somewhere around here I think we need to add a few sentences or point to the existing documentation to point out that the user running this needs to spin up a local docker-compose environment for this to work. Submitting the job requires a localhost:6379 be present to work.